geoportal/c2cgeoportal_geoportal/scaffolds/create/scripts/db-backup

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021, Camptocamp SA
+# All rights reserved.
+
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions and the following disclaimer in the documentation
+#    and/or other materials provided with the distribution.
+
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+# ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+# (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+# ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+# The views and conclusions contained in the software and documentation are those
+# of the authors and should not be interpreted as representing official policies,
+# either expressed or implied, of the FreeBSD Project.
+
+
+import argparse
+import subprocess
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(description="Backup the remote database.")
+    parser.add_argument(
+        "--env", default=".env", help="The env file to use to get the connection settings, by default .env."
+    )
+    parser.add_argument("--arg", action="append", help="Additional pg_dump argument")
+    parser.add_argument("file", help="The file path to backup")
+    args = parser.parse_args()
+
+    env = {}
+    with open(args.env) as f:
+        for line in f:
+            if line and line[0] != "#":
+                try:
+                    index = line.index("=")
+                    env[line[:index].strip()] = line[index + 1 :].strip()
+                except ValueError:
+                    # Ignore lines that don't have a '='
+                    pass
+
+    with open(args.file) as f:
+        subprocess.check_call(
```

Comment:
`subprocess.run` is recommended when possible. Here we can use it with `check=True`